File: services/reading_mail/reading_mail.js
const superagent = require('superagent')
const mailsac_api_key = 'YOUR_API_KEY_HERE'

superagent
  .get('https://mailsac.com/api/addresses/calendartrinity@mailsac.com/messages')
  .set('Mailsac-Key', mailsac_api_key)
  .then((messages) => {
      messageId = messages.body[0]._id
      superagent
          .get('https://mailsac.com/api/text/calendartrinity@mailsac.com/' + messageId)
          .set('Mailsac-Key', mailsac_api_key)
              .then((messageText) => {
                  console.log(messageText.text)
              })
  })
  .catch(err => {
      console.log(err.message)
  })

/*
Dogtreats are good!
*/

File: services/reading_mail/reading_mail.py
import requests
from pprint import pprint

headers = {'Mailsac-Key': 'YOUR_API_KEY_HERE'}
url = 'https://mailsac.com/api/addresses/calendartrinity@mailsac.com/messages'
r = requests.get(url, headers=headers)
message_id =  r.json()[0]['_id']
url = 'https://mailsac.com/api/text/calendartrinity@mailsac.com/' + message_id
r = requests.get(url, headers=headers)
pprint(r.text)

'''
'Dogteats are good!'
'''
